Source/WebInspectorUI/UserInterface/Controllers/JavaScriptRuntimeCompletionProvider.js

```diff
@@ -91,7 +91,17 @@ WI.JavaScriptRuntimeCompletionProvider = class JavaScriptRuntimeCompletionProvid
         return JavaScriptRuntimeCompletionProvider.__cachedCommandLineAPIKeys;
     }
 
-    // Protected
+    // Public
+
+    clearCachedPropertyNames()
+    {
+        if (this._clearCachedPropertyNamesTimeout) {
+            clearTimeout(this._clearCachedPropertyNamesTimeout);
+            this._clearCachedPropertyNamesTimeout = null;
+        }
+
+        this._lastPropertyNames = null;
+    }
 
```

Comment:
NIT: Add a `// Protected` here, unless are we making the rest public for some reason.

Reply:
Ah yes, `clearCachedPropertyNames()` was meant to be public, and I removed the `// Protected` comment since the function below that it used to mark as protected, `completionControllerCompletionsNeeded()`, is treated as public and called by others anyway (such as [`CodeMirrorCompletionController`](https://github.com/WebKit/WebKit/blob/b3eceeb22f58375c3ae39ccdc376335e417881fe/Source/WebInspectorUI/UserInterface/Controllers/CodeMirrorCompletionController.js#L533)), so I thought marking it as protected would be misleading 😊